src/_designs/screens/dashboard/index.vue

```diff
@@ -1,8 +1,13 @@
 <template>
   <div class="app-dashboard">
-    <PageHeader title="Dashboard">
-      <BreadCrumbs />
-    </PageHeader>
+    <div class="graphs-container">
+      <div class="graph">
+        <canvas id="transaction-rate" />
```

Comment:
its early days now, but can you make a scheme on how to componetize these, maybe based on the work already done on other components